ChuckNorrisFacts/Presentation/List Facts/ListFactsViewModel.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  ListFactsViewModel.swift
+//  ChuckNorrisFacts
+//
+//  Created by Munir Wanis on 21/05/18.
+//  Copyright © 2018 Wanis. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+
+class ListFactsViewModel {
+    let service = FactService()
+    
+    var presentation: Observable<[FactPresentation]>
+    
+    init() {
+        presentation = service.getFacts(term: "")
```

Comment:
Por que de início vc faz um request com o termo vazio? 🤔 Ele retorna um erro assim, não retorna?

Reply:
Era pra teste, no PR da etapa 4 não estará mais assim.